src/components/v2/TokenTextField/FormikTokenTextField.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useField } from 'formik';
+import { TokenTextField, ITokenTextFieldProps } from '.';
+
+interface IFormikTokenTextField extends Omit<ITokenTextFieldProps, 'name' | 'onChange' | 'value'> {
+  name: string;
+  errorCodes?: string[];
```

Comment:
I think the name `errorCodes` is confusing in this context, as I'd expect it to correspond to the codes of all the current errors for this component.
Perhaps `displayableErrorCodes` or something like that?